metadata.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "razorsedge-openvmtools",
-  "version": "1.0.0",
+  "version": "1.0.1",
```

Comment:
Please do not increase the version number. I will do that as a part of the module release process.